update.sh

```diff
@@ -57,22 +65,22 @@ if [[ ! -f "$HOME/.rhino/config/5-18-11" ]]; then
     echo "Verifying checksums..."
     if shasum --check --ignore-missing CHECKSUMS; then
       sudo apt install ./*.deb
-      : > "$HOME/.rhino/config/5-18-11"
+      : > "/usr/share/rhino/config/5-18-11"
```

Comment:
Requires sudo?

Reply:
actually no but you can run rhino-update as sudo. the user has permissions to read/write to that directory. I used chmod/chown at the end of the script.

Reply:
I see. so I can mark all of them as resolved?

Reply:
yes you can.